x/projects/keeper/creation.go

```diff
@@ -0,0 +1,79 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/lavanet/lava/utils"
+	"github.com/lavanet/lava/x/projects/types"
+)
+
+// add a default project to a subscription
+func (k Keeper) CreateDefaultProject(ctx sdk.Context, subscriptionAddress string) error {
+	project := types.DefualtProject(subscriptionAddress) // TODO add the CU per epoch here
+	var emptyProject types.Project
+
+	err := k.projectsFS.FindEntry(ctx, project.Index, uint64(ctx.BlockHeight()), &emptyProject)
+	if err == nil {
+		return utils.LavaError(ctx, ctx.Logger(), "CreateDefaultProject_already_exist", map[string]string{"subscription": subscriptionAddress}, "default project already exist for the current subscription")
+	}
+
+	project.Enabled = true
+	// add subscription key as developer key to the default project
+	projectID := types.ProtoString{String_: project.Index}
+	k.developerKeysFS.AppendEntry(ctx, project.Index, uint64(ctx.BlockHeight()), &projectID)
+
+	return k.projectsFS.AppendEntry(ctx, project.Index, uint64(ctx.BlockHeight()), &project)
+}
+
+// add a new project to the subscription
+func (k Keeper) CreateEmptyProject(ctx sdk.Context, subscriptionAddress string, projectName string, adminAddress string, enable bool) error {
+	project := types.CreateEmptyProject(subscriptionAddress, projectName)
+	var emptyProject types.Project
+
+	err := k.projectsFS.FindEntry(ctx, project.Index, uint64(ctx.BlockHeight()), &emptyProject)
+	// the project with the same name already exists if no error has returned
+	if err == nil {
+		return utils.LavaError(ctx, ctx.Logger(), "CreateEmptyProject_already_exist", map[string]string{"subscription": subscriptionAddress}, "project already exist for the current subscription with the same name")
+	}
+
+	project.ProjectKeys = append(project.ProjectKeys, types.ProjectKey{Key: adminAddress, Types: []types.ProjectKey_KEY_TYPE{types.ProjectKey_ADMIN}})
+
+	var projectID types.ProtoString
```

Comment:
refactor to use a helper `project.AddKey()` (e.g. in x/project/types/project.go)